mission1/style.css

```
@@ -0,0 +1,354 @@
+@font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Thin.woff') format('woff');
+    font-weight: 100;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-ExtraLight.woff') format('woff');
+    font-weight: 200;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Light.woff') format('woff');
+    font-weight: 300;
+    font-style:normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
+    font-weight: 400;
+    font-style:normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Medium.woff') format('woff');
+    font-weight: 500;
+    font-style:medium;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-SemiBold.woff') format('woff');
+    font-weight: 600;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Bold.woff') format('woff');
+    font-weight: 700;
+    font-style:bold;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-ExtraBold.woff') format('woff');
+    font-weight: 800;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Black.woff') format('woff');
+    font-weight: 900;
+    font-display: swap;
+}
+
+
+*{
+  box-sizing:border-box;
+  padding:0;
+  margin:0;
+  word-break: keep-all;
+  color:#374151;
+}
+
+ul,ol,li{
+  list-style:none;
+  margin:0;
+  padding:0;
+}
+
+body{
+  font-family:'Pretendard';
+  font-size:1.6rem;
+  line-height:1.5;
+  transition: font-size 0.3s ease;
+}
+
+a{
+  text-decoration:unset;
+}
+
+img{
+  max-width:100%;
+}
+
+
+html{
+  font-size: 62.5%; /*브라우저 기본 16px x 0.625 = 10px */
+  transition:font-size 0.3s ease;
+}
+
+@media (min-width: 375px) and (max-width:767px){

```

💊 제안
media query 문은 일반적으로 하단에 적습니다.
지금 요한님께서는 조건을 한정하셔서 상관없지만 css 에서 추후 선언된 스타일이 우선순위를 가지기 때문에 반응형 관련 css들을 하단에 적어 같은 선택자 우선순위더라도 잘 적용될 수 있게 하기 위해서입니다~
또한 미디어 쿼리가 적용되는 조건을 375~767같이 한정하시게 되면 css를 상속받지 못하고 예상대로 동작하지 않을 수 있으니 아래처럼 조건을 변경해주시고 하단에 적는 것을 추천드립니다.

```css
// case1
.classname { background-color: red }
@media (max-width: 800px) {
    .classname { background-color: blue } // 미디어 쿼리의 조건을 만족시키면 해당 사항이 적용.
}

// case2
@media (max-width: 800px) {
    .classname { background-color: blue } 
}
.classname { background-color: red } // 무조건 해당 사항이 적용
```

```suggestion
/** 
min-width가 조건에 걸려있어 375px 미만으로 화면이 작아지게 되면 html에 font-size: 62.5%이 적용되므로
의도대로 동작하지 않을 수 있습니다.
*/
@media (max-width:767px){
```